### DISCUSSION TITLE
I want to create a pdf with a background image and a string as title, any sample code?

### DISCUSSION THREAD

**Discussion by nissansz** [pymupdf/PyMuPDF#2043] [Q&A]
I want to create a pdf with a background image and a string as title, any sample code?

**Comment by JorjMcKie** [comment]
Start with `page.insert_image()` and `page.insert_text()`. Lots of example also in [this](https://github.com/pymupdf/PyMuPDF-Utilities) repo.

**Comment by nissansz** [comment]
Thank you.
